components/prism-javascript.js

```diff
@@ -19,16 +19,22 @@ Prism.languages.insertBefore('javascript', 'keyword', {
 	},
 	'parameter': [
 		{
-			pattern: /(function)(?:\s+[_$a-z\xA0-\uFFFF][$\w\xA0-\uFFFF]*)?\s*(\([^()]*\))/,
+			pattern: /(function(?:\s+[_$a-z\xA0-\uFFFF][$\w\xA0-\uFFFF]*)?\s*\(\s*)\S[^()]*(?=\s*\))/,
```

Comment:
2 Problems here:

1. You probably don't want trailing spaces, so please make it `[^()]*?`. Otherwise `(?=\s*` won't match anything.
2. This also matches `function a())`. Not valid JS, I know, but still. Replace `\S` with `[^\s()]` and your good to go.